acceptance-tests/remote.go

```diff
@@ -0,0 +1,236 @@
+package acceptance_tests
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+	"time"
+
+	"github.com/bramvdbogaerde/go-scp"
+	"golang.org/x/crypto/ssh"
+)
+
+// runs command on remote machine
```

Comment:
please adapt to a proper godoc comment.

Applies to all functions in this file.

Reply:
ok, will do. Can you open an issue with HAProxy bosh release to do the same there? Most of this code was copied over.

Reply:
done. https://github.com/cloudfoundry/haproxy-boshrelease/issues/482